new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/Pod.java

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes.actions.impl;
+
+import io.kubernetes.client.openapi.ApiException;
+import io.kubernetes.client.openapi.models.V1Pod;
+import io.kubernetes.client.openapi.models.V1PodList;
+import oracle.weblogic.kubernetes.actions.impl.primitive.Kubernetes;
+
+public class Pod {
+
+  /**
+   * Delete Kubernetes Pod.
+   *
+   * @param name name of the pod
+   * @param namespace name of namespace
+   * @return true if successful, false otherwise
+  */
+  public static boolean deletePod(String name, String namespace) {
+    return Kubernetes.deletePod(name, namespace);
+  }
+
+  /**
+   * List Kubernetes Pod(s) in a namesapce.
+   *
+   * @param namespace name of namespace
+   * @param labelSelectors with which pods are decorated
+   * @return V1PodList list of pods
+  */
+  public static V1PodList listPods(String namespace, String labelSelectors) {
+    V1PodList retPodList = null;
+    try {
+      retPodList = Kubernetes.listPods(namespace,labelSelectors);
+    } catch (ApiException api) {
```

Comment:
do not catch the exception here.

Reply:
Removed all the API exception catch in Pod.java